wave_matching/tests/ndt_tests.cpp

```diff
@@ -5,8 +5,8 @@
 
 namespace wave {
 
-#define TEST_SCAN "tests/data/testscan.pcd"
-#define TEST_CONFIG "tests/config/ndt.yaml"
+#define TEST_SCAN "data/testscan.pcd"
+#define TEST_CONFIG "config/ndt.yaml"
```

Comment:
probably want to do a `static const std::string TEST_CONFIG = "config/ndt.yaml"`

The reason is because the `#define TEST_CONFIG` is defined in other test files, this is working without modification because this line overwrites the previous `TEST_CONFIG` macro. Ideally we want to: 

1. avoid the use of macros (leo will bring this up, as he already has with #84 [example](https://github.com/wavelab/libwave/blob/master/wave_vision/tests/fast_tests.cpp)). 
2. avoid name conflicts

One solution is to use `static` to limit the scope of the variable to only this file, and `const` because we're going to set it once and never change again. Hence the

    static const std::string TEST_CONFIG = "config/ndt.yaml"